cli/tests/config

```diff
@@ -0,0 +1,16 @@
+[aws]
+aws_region_name = us-east-1
+
+[cluster default]
+vpc_settings = public
+key_name = test
+
+[vpc public]
+master_subnet_id = subnet-2ce814d3
```

Comment:
Do these actually get used?

Reply:
Yes, they are written and used. This is only used for the sections other than [vpc public]